modules/core/shared/src/main/scala/lucuma/core/enums/ConvenienceOps.scala

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.enums
+
+import lucuma.core.math.BoundedInterval
+import lucuma.core.math.Wavelength
+
+private[enums] trait ConvenienceOps {
+  extension (i: Int) {
+    def pm: Wavelength =
+      Wavelength.unsafeFromIntPicometers(i)
+
+    def gePmRange: BoundedInterval[Wavelength] =
```

Comment:
What does the `ge` stand for?

Reply:
greater than or equal.  Makes a range range from `[n, Int.MaxValue)`.  It might logically be `[n, Int.MaxValue]` but looking ahead, the Postgres `int4range` can't handle that.